### PR TITLE
Bugfix: Prevent TS error on CheckboxGroupInput label styles

### PR DESCRIPTION
**Pull request by andrico1234**
Closes https://github.com/marmelab/react-admin/issues/5985

https://user-images.githubusercontent.com/17087167/110091540-75cc0680-7d90-11eb-9086-3b9d86d3b1d7.mov

### CODE REVIEW

File: packages/ra-ui-materialui/src/input/CheckboxGroupInput.tsx [hunk 1]
```diff
@@ -80,7 +81,8 @@ import { LinearProgress } from '../layout';
  * The object passed as `options` props is passed to the material-ui <Checkbox> components
  */
 const CheckboxGroupInput: FunctionComponent<
-    ChoicesInputProps<CheckboxProps> & FormControlProps
+    ChoicesInputProps<CheckboxProps> &
+        FormControlProps & { classes?: ClassesOverride<typeof useStyles> }
```
Comment (djhi) [inline packages/ra-ui-materialui/src/input/CheckboxGroupInput.tsx:85]: At this point, I think we should extract this into a `CheckboxGroupInputProps` interface and export it (don't forget the index file)

**Review by djhi — CHANGES_REQUESTED** [review]
Thanks for working on this!

File: packages/ra-ui-materialui/src/input/CheckboxGroupInput.tsx (hunk 1 above)
Comment (andrico1234) [reply packages/ra-ui-materialui/src/input/CheckboxGroupInput.tsx]: Done

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**